OPHD/UI/Core/ListBox.cpp

```diff
@@ -169,7 +169,7 @@ const ListBox::ListBoxItem& ListBox::selected() const
 void ListBox::clear()
 {
 	mItems.clear();
-	mSelectedIndex = 0;
+	mSelectedIndex = constants::NO_SELECTION;
```

Comment:
Definitely a potential bug here. I think other parts of the code sort of protect against this but this is definitely a defect.